src/main/java/org/gridsuite/voltageinit/server/service/VoltageInitService.java

```diff
@@ -43,6 +48,18 @@ public UUID runAndSaveResult(VoltageInitRunContext runContext) {
         return resultUuid;
     }
 
```

Comment:
@ Transactional(readOnly = true) ?    because the annotation just on thenfind method may be not sufficient ?

Reply:
Done